content/doc/book/managing/cli.adoc

```diff
@@ -185,14 +185,22 @@ Authorities:
 %
 ----
 
-
 == Using the CLI client
 
 While the SSH-based CLI is fast and covers most needs, there may be situations where the CLI
 client distributed with Jenkins is a better fit. For example, the default transport for the CLI client
 is HTTP which means no additional ports need to be opened in a firewall for its
 use.
 
+=== Comparing SSH and CLI client
+
+Both SSH and jenkins-cli.jar provide access to a set of commands that lets you interact with Jenkins from a command line, but they have a few differences:
+
+* Jenkins SSH does not require any custom jar file on the client side, making it easier to access Jenkins from a variety of sources
+* SSH client was build to be a generic tool to serve several purposes.
+It doesn't offer an easy way to do basic things that are common and specific to Jenkins environments
+Using the `jenkins-cli.jar` instead of the ssh client may increase productivity and improve the development experience
+
```

Comment:
```suggestion
* Jenkins SSH does not require any custom jar file on the client side, making it easier to access Jenkins from a variety of sources.
* SSH client was build to be a generic tool to serve several purposes.
It doesn't offer an easy way to do basic things that are common and specific to Jenkins environments.
Using the `jenkins-cli.jar` instead of the ssh client may increase productivity and improve the development experience.

```